lib/project_types/php/messages/messages.rb

```diff
@@ -6,8 +6,43 @@ module Messages
       php: {
         create: {
           help: <<~HELP,
-            {{command:%s create php}}: Not implemented yet
+          {{command:%s create php}}: Creates an embedded PHP app.
+            Usage: {{command:%s create php}}
+            Options:
+              {{command:--name=NAME}} App name. Any string.
+              {{command:--organization-id=ID}} Partner organization ID. Must be an existing organization.
+              {{command:--shop-domain=MYSHOPIFYDOMAIN }} Development store URL. Must be an existing development store.
+              {{command:--type=APPTYPE }} Whether this app is public or custom.
+              {{command:--verbose }} Output verbose information when installing dependencies.
           HELP
+
+          error: {
+            php_required: "PHP is required to create an app project. Visit https://www.php.net/manual/en/install.php " \
```

Comment:
```suggestion
            php_required: "PHP is required to create an app project. Visit {{underline:https://www.php.net/manual/en/install.php}} " \
```
and same with other links

Reply:
And I **believe** we want links to be on a new line as much as possible, so rewording things like
```suggestion
            php_required: <<<INSTALL
            PHP is required to create an app project. For installation instructions, visit: 
              {{underline:https://www.php.net/manual/en/install.php}}
            INSTALL
```